src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -5544,7 +5559,7 @@ arch_emit_simd_intrinsics (const char *class_ns, const char *class_name, MonoCom
 	}
 
 	if (!strcmp (class_ns, "System.Runtime.Intrinsics")) {
-		if (!strcmp (class_name, "Vector128`1"))
+		if (!strcmp (class_name, "Vector128`1") || !strcmp (class_name, "Vector64`1"))
```

Comment:
Just noting that `Vector64` isn't accelerated on x86/x64 for RyuJIT as its not a type that has native support.

There's the legacy `__m64` support via MMX which uses the x87 FPU stack and is deprecated
so to get "good support" you'd have to fill half a V128 and mask off the upper bits for various operations (such as `Sum` and `DotProduct`)

It's fine to do that, just might be more effort than desirable overall. If Mono does do it here, then it'd be good to log an issue tracking RyuJIT getting similar support so we have parity on what users can expect.

Reply:
Will disable support for it then.